src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns the dashboard page. */
+@WebServlet("/")
+public class HomeServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+    response.sendRedirect("creatives/maps/dashboard.html");
```

Comment:
If sendRedirect works, doesn't that mean that the user can just go directly to <baseurl>/creatives/maps/dashboard.html to see the page?

Reply:
Yes. I realized that halfway through but I thought the home route taking them there anyway would be easier for them to use. What do you think?

Reply:
Would it be better to move the js file associated with it too? or just the html file is fine?